backend/backend.go

```diff
@@ -540,6 +544,97 @@ func (backend *Backend) Accounts() AccountsList {
 	return backend.accounts
 }
 
+// AccountsByKeystore returns a map of the current accounts of the backend, grouped
+// by keystore.
+func (backend *Backend) AccountsByKeystore() (KeystoresAccountsListMap, error) {
```

Comment:
i wonder if this could replace the `getAccountsByKeystore()` frontend function which basically does the same.

Reply:
I had a look but I think that requires a bit of rework and deserves a separate PR, since the accounts are fetched in app.tsx and then passed as properties to the child components. Maybe something for @shonsirsha ? :)